packages/contracts/contracts/foundry/Sphinx.sol

```diff
@@ -314,21 +314,21 @@ abstract contract Sphinx {
             "Sphinx: Cannot call Sphinx using vm.prank. Please use vm.startPrank instead."
         );
 
-        // We allow users to call `vm.startPrank` before calling their `deploy` function so that
-        // they don't need to toggle it before and after calling `deploy`, which may be annoying for
+        // We allow users to call `vm.startPrank` before calling their entry point function so that
+        // they don't need to toggle pranking before and after calling it, which may be annoying for
         // users who have complex deployment flows. However, we turn pranking off here because we'll
-        // prank the Gnosis Safe during the execution process, since this is the contract that
+        // broadcast from the Gnosis Safe during the execution process, since this is the contract that
         // deploys their contracts on live networks. If the user enabled pranking before calling
-        // `deploy`, then we'll turn it back on at the end of this modifier.
+        // their entry point function, then we'll turn it back on at the end of this modifier.
         if (callerMode == VmSafe.CallerMode.RecurrentPrank) vm.stopPrank();
 
         sphinxUtils.fetchAndValidateConfig(address(this));
 
-        // Prank the Gnosis Safe then execute the user's script. We prank the Gnosis
-        // Safe to replicate the production environment.
-        vm.startPrank(safeAddress());
+        // Broadcast from the Gnosis Safe then execute the user's script. We broadcast from the
+        // Gnosis Safe to replicate the production environment.
+        vm.startBroadcast(safeAddress());
```

Comment:
Broadcasting instead of pranking solves the [contract address miscalculation ticket](https://linear.app/chugsplash/issue/CHU-659/contract-addresses-not-properly-calculated-for-new-keyword)